apps/wallet/src/dapp-interface/WalletStandardInterface.ts

```diff
@@ -282,6 +316,55 @@ export class SuiWallet implements Wallet {
 		);
 	};
 
+	#signAndExecuteTransactionBlockV2: SuiSignAndExecuteTransactionBlockV2Method = async (input) => {
+		return mapToPromise(
+			this.#send<ExecuteTransactionRequest, ExecuteTransactionResponse>({
```

Comment:
this is adding a simplified implementation of `signAndExecuteTransactionBlockV2` that **MOSTLY** uses the existing behavior.

The big difference here is that we are sending the payload in the new JSON format rather than the old one.  This shouldn't require any changes on the wallet changes, since it will re-construct the txb using `TransactionBlock.from` which works with both the old and new JSON formats.

In the future, we should update `'execute-transaction-request'` to always return signature/bytes/effects/balanceChanges rather than relying on the show* options, but threading that through is a more complex change I didn't want to make in this PR